source/_integrations/zwave_js.markdown

```diff
@@ -1016,6 +1016,24 @@ Switching does not require renaming your devices.
 7. Uninstall the official **Z-Wave JS** add-on.
    - You are asked if you want to delete the related data. Keep it if you think you might switch back to the **Z-Wave JS** add-on later.
 
+### How to migrate from one adapter to a new adapter using Z-Wave JS UI?
+
+If you are currently using [Z-Wave JS UI](https://zwave-js.github.io/zwave-js-ui/#/) instead of the official **Z-Wave JS** add-on and want to start using a new adapter, you can migrate your network inside the **Z-Wave JS UI** add-on.
+
+1. Before starting migration, disable the **Z-Wave** integration.
+   - Go to {% my integrations title="**Settings** > **Devices & services**" %} and select the Z-Wave integration and select the three dots {% icon "mdi:dots-vertical" %} menu and select **Disable**.
+2. Do the migration in Z-Wave JS UI.
+   - If you are using the **Z-Wave JS UI** add-on, go to {% my supervisor_addon addon="core_zwave_jsa0d7b954_zwavejs2mqtt" title="**Settings** > **Add-ons** > **Z-Wave JS UI**" %}
+   - Open the Z-Wave JS UI control panel and in the bottom-right corner, select the purple **Advanced actions** button.
+   - Under **NVM Management**, select **Backup**.
+   - Unplug the current adapter and connect the new adapter.
+   - Under **Settings** > **UI** > **Z-Wave**, enter the region and save.
```

Comment:
I think we should mention that you may need to update the device path in Z-Wave JS UI after connecting the new adapter. Preferably you should use the serial-by-id device path which will normally not be the same when changing adapter model.